blocks/core/ff_module/ff_module-task-event-repeater/ff_module-task-event-repeater.rt

```diff
@@ -0,0 +1,11 @@
+<rt-require dependency="../../ff_module/ff_module-task-event/ff_module-task-event.js" as="TaskEvent"/>
+
+<div class="ff_container-task-event-repeater">
+    <ol class="ff_container-task-event-repeater__items">
+        <li class="ff_container-task-event-repeater__item"
+            key= "event-{eventIndex}"
```

Comment:
I think it's fine to assume that they're passed in chronological order, so this should just render them in reverse order. (Although I find the reverse order a bit confusing -- I assume others have already thought about this though?)

Each event has a local ID, so we could pass that through as a property on the event.